tasks/hickerson-series-of-almost-integers/src/main.rs

```diff
@@ -0,0 +1,29 @@
+extern crate factorial;
```

Comment:
Because you are using the 2018 edition, [you don't need to declare `extern crate`](https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html?highlight=extern#no-more-extern-crate). So you can remove both these lines! :raised_hands:

Reply:
done :)